iot-hub/Samples/device/PnpDeviceSamples/Thermostat/Program.cs

```diff
@@ -43,9 +43,10 @@ public static async Task Main(string[] args)
                 });
             s_logger = loggerFactory.CreateLogger<Program>();
 
+            // Instantiating this seems to do all we need for outputting SDK events to our console log.
+            // The SDK logs are written at Trace level. To view SDK logs change the MinLogLevel to Trace.
             const string SdkEventProviderPrefix = "Microsoft-Azure-";
-            // Instantiating this seems to do all we need for outputting SDK events to our console log
-            _ = new ConsoleEventListener(SdkEventProviderPrefix, s_logger);
+            _ = new ConsoleEventListener(SdkEventProviderPrefix, logger);
```

Comment:
similar to [this ](https://github.com/Azure-Samples/azure-iot-samples-csharp/pull/274#discussion_r864292360)